### PR TITLE
Update fr.m3u

### PR DESCRIPTION
**Pull request by trancer1994**

### CODE REVIEW

File: channels/fr.m3u [hunk 1]
```diff
@@ -337,3 +337,63 @@ https://streamer03.myvideoplace.tv/streamer02/hls/MDS_VIA_PAD_301117_medium/inde
...
+#EXTINF:-1 tvg-id="" tvg-name="Chérie 25" tvg-country="" tvg-language="" tvg-logo="https://upload.wikimedia.org/wikipedia/fr/thumb/f/f0/Ch%C3%A9rie_25_logo_2015.svg/51px-Ch%C3%A9rie_25_logo_2015.svg.png" group-title="",Chérie 25
+http://cdn.webtv4.cdnfr.orange.fr/hs/HALO3/hls/cherie25-850448/hls/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i2.wp.com/www.millereponses.com/wp-content/uploads/2018/11/chaine-nord-africaine-frequence-nilesat.jpg" group-title="",Chaine Nord Africaine
+http://live.creacast.com/cna/smil:cna.smil/chunklist_w1780122037_b864000.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Chaine Nord-Africaine
+http://str81.creacast.com/cna/live.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Citruss TV
+https://citrusstv.akamaized.net/hls/live/687285/CTV/01.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CNA
+https://d2e1asnsl7br7b.cloudfront.net/7782e205e72f43aeb4a48ec97f66ebbe/index_2.m3u8
```
Comment (freearhey) [inline channels/fr.m3u:349]: CNA is a Singapore channel: https://www.lyngsat.com/tvchannels/sg/CNA.html

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/fr.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/fr.m3u:349]: Is CNA another name for "Chaîne Nord Africaine" ? I should watch the live and see by myself.

**Comment by freearhey** [comment]
Closed due to lack of any activity.

**Pull request closed without merging**